src/selectors/__specs__/selectVenuesByOffererIdAndOfferType.spec.js

```diff
@@ -0,0 +1,42 @@
+import selectVenuesByOffererIdAndOfferType from '../selectVenuesByOffererIdAndOfferType'
+
+describe('src | selectors | selectVenuesByOffererIdAndOfferType', () => {
+  it('should return an empty array when state contains no venues', () => {
+    // given
+    const state = {
+      data: {
+        venues: [],
+      },
+    }
+
+    // when
+    const result = selectVenuesByOffererIdAndOfferType(state)
+
+    // then
+    expect(result).toStrictEqual([])
+  })
+
+  it('should return array of object when state contains on venue that is virtual', () => {
+    // given
+    const state = {
+      data: {
+        venues: [
+          {
+            isVirtual: true,
+          },
+        ],
+      },
+    }
+
+    // when
+    const expected = [
+      {
+        isVirtual: true,
+      },
+    ]
+    const result = selectVenuesByOffererIdAndOfferType(state)
+
+    // then
+    expect(result).toStrictEqual(expected)
```

Comment:
```suggestion
    expect(result).toStrictEqual([
      {
        isVirtual: true,
      },
    ])
```
Car ton `when` ne doit contenir que ton intention.

Reply:
je trouve ça moins lisible que `expect(result).toStrictEqual(expected)`, je préfère redescendre le expected dans le then